src/pages/List/List.js

```diff
@@ -1,8 +1,68 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import Product from './Product';
 import './List.scss';
+import { useSearchParams } from 'react-router-dom';
 
 const List = () => {
-  return <p className="">List</p>;
+  const [products, setProducrts] = useState([]);
```

Comment:
```suggestion
  const [products, setProducts] = useState([]);
```
오타도 유지보수에 치명적인 악영향을 미칠 수 있기 때문에 꼼꼼하게 확인해주세요!